braindecode/datautil/transforms.py

```diff
@@ -77,103 +75,20 @@ def _transform(raw_or_epochs, transforms):
             getattr(raw_or_epochs.load_data(), transform)(**transform_kwargs)
 
 
-class FilterRaw(object):
-    """Apply mne filter on raw data
-
-    Parameters
-    ----------
-    mne_filter_kwargs : **kwargs
-        kwargs passed to mne.io.Raw.filter
-    """
-
-    def __init__(self, **mne_filter_kwargs):
-        self.mne_filter_kwargs = mne_filter_kwargs
-
-    def __call__(self, raw):
-        """Apply filter
-
-        Parameters
-        ----------
-        raw : mne.io.Raw
-            raw data to filter
-        """
-        return raw.filter(**self.mne_filter_kwargs)
-
-
-class ZscoreRaw(object):
-    """Zscore raw data channel wise
-    """
-
-    def __call__(self, raw):
-        """Zscore Normalize raw data channel wise
-
-        Parameters
-        ----------
-        raw : mne.io.Raw
-            raw data to normalize
-
-        Returns
-        -------
-        raw : mne.io.Raw
-            normalized raw data
-
-        """
-        raw = raw.apply_function(lambda x: x - x.mean())
-        return raw.apply_function(lambda x: x / x.std())
-
-
-class FilterWindow(object):
-    """FIR filter for windowed data.
+def zscore(continuous_data):
```

Comment:
I think this function should also work with windowed data, we might just need to use `keepdims=True`. If that's desirable, then the argument could be renamed to `data`.

Reply:
Yes, good idea. So transforms should work on numpy ndarrays. For application to mne.Raw we pass them to <code>raw.apply_function()</code>. For application to mne.Epochs there is no such function. (Could mne implement such a function?) We would then implement a way to get the ndarrays from the Epochs object, transform it, and put it pack. This will likely take place inside the Dataset class as in torchvision.